qemu/tests/cfg/blockdev_stream_with_ioerror.cfg

```diff
@@ -0,0 +1,52 @@
+- blockdev_stream_with_ioerror:
+    type = blockdev_stream_with_ioerror
+    virt_test_type = qemu
+    images += " data"
+    force_create_image_data = yes
+    force_remove_image_data = yes
+    start_vm = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    base_tag = "data"
+    image_size_data = 2G
+    snapshot_tag = sn1
+    image_size_sn1 = 2G
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
```

Comment:
set image_format_data = qcow2 for only qcow2 can be created in the space whose size is less than the qcow2 image